spec/core_functions/color/same.hrx

```diff
@@ -0,0 +1,28 @@
+<===> known_space/input.scss
+a {b: same(hsl(0 100% 25.1%), 'red')}
```

Comment:
```suggestion
@use 'sass:color';
a {b: color.same(hsl(0 100% 25.1%), 'red')}
```
This function is not global. So you have to call it from the `sass:color` module.